features/order/managing_orders/cancelling_order_with_payment.feature

```diff
@@ -0,0 +1,22 @@
+@managing_orders
+Feature: Checking payment state of a cancelled order
+    In order to check payment state after cancelling an order
+    As an Administrator
+    I want to payment has proper state
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And there is a customer "john.doe@gmail.com" that placed an order "#00000022"
+        And the customer bought a single "PHP T-Shirt"
+        And the customer chose "Free" shipping method to "France" with "Cash on Delivery" payment
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Checking payment state of a placed order
+        When I view the summary of the order "#00000022"
+        And I cancel this order
+        Then I should be notified that it has been successfully updated
+        Then it should have cancelled payment state
```

Comment:
I wasn't sure if we should join these checks into one scenario, but I of course agree with you. It would be more readable.